README.md

```
@@ -39,164 +28,122 @@ npm install
+  id uuid PRIMARY KEY,
+  task_id uuid REFERENCES tasks(id),
+  voter_name text NOT NULL,
+  choice text CHECK (choice IN ('yes', 'no', 'maybe')),
+  UNIQUE(task_id, voter_name)
+);
 ```
 
-## 📋 Project Structure
+## Project Structure
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)